Build-qt.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
```

Comment:
As suggested [here](https://github.com/KitwareMedical/TubeTK-DashboardScripts/pull/88#discussion_r71071092), what do you think if the script is named `linux_build_qt.sh` ?